fastreplacestring/config/discover.ml

```diff
@@ -0,0 +1,40 @@
+module C = Configurator.V1
+
+let () =
+  C.main ~name:"configure fastreplacestring" (fun c ->
+
+    let flags =
+      match C.ocaml_config_var c "system" with
+      | Some "mingw64" -> [
+          "-ccopt"; "-lstdc++";
+        ]
+      | Some _
+      | None -> [
+          "-ccopt"; "-Ofast";
+          "-ccopt"; "-lstdc++";
+        ]
+    in
+    C.Flags.write_sexp "dune.flags" flags;
+
+    let cflags = ["-lstdc++"] in
+    C.Flags.write_sexp "dune.cflags" cflags;
+
+    let cxx_flags =
+      match C.ocaml_config_var c "system" with
+      (* link statically on windows so we don't have to ship mingw64.dll *)
+      | Some "mingw64" -> [
+        "-static";
+        "-static-libgcc";
+        "-static-libstdc++";
```

Comment:
Along with the investigation I mentioned in Discord, I also tried setting `-fno-rtti` and `-fno-exceptions` here, but didn't have any luck.